### PR TITLE
Fix Solars with Silicate

### PR DESCRIPTION
**Pull request by Kurfursten**
🆑 
* rscadd: You can now fix damaged solars by spraying them with silicate.

### CODE REVIEW

File: code/modules/power/solars/panel.dm [hunk 1]
```diff
@@ -349,3 +349,24 @@
...
+		return 1
+
+	diff = min(diff, S.get_amount() / SILICATE_PER_DAMAGE)
+
+	health += diff
+	healthcheck(user, FALSE)
+
+	user.visible_message("<span class='notice'>[user] repairs \the [src] with their [S]!</span>", "<span class='notice'>You repair \the [src] with your [S].</span>")
+
+	playsound(src, 'sound/effects/refill.ogg', 10, 1, -6) //Probably will never hear this!
```
Comment (DeityLink) [inline code/modules/power/solars/panel.dm:374]: I mean technically solar panels work just fine indoors.

**Comment by DeityLink** [comment]
I'm of the opinion that this should only repair "damaged" solars, not "broken" ones, which should still require a glass replacement.
image

File: code/modules/power/solars/panel.dm (hunk 1 above)
Comment (Kurfursten) [reply code/modules/power/solars/panel.dm:374]: Just an observation - probably

**Comment by Kurfursten** [comment]
That's fine. "Broken" solars always have exactly 1 HP, right?

**Pull request merged**